crates/core/tedge_mapper/Cargo.toml

```diff
@@ -25,6 +25,7 @@ collectd_ext = { workspace = true }
 flockfile = { workspace = true }
 mqtt_channel = { workspace = true }
 reqwest = { workspace = true }
+rumqttc = { workspace = true }
```

Comment:
I would avoid to add this direct dependency to `rumqttc`.

As this is done only to build an `rumqttc::MqttOptions` struct passed to `MqttBridgeActorBuilder`, can this dependency hidden behind a light facade by `tedge_mqtt_bridge`?

Reply:
Changed to a `pub use` in `tedge_mqtt_bridge` and then importing from there